lib/standalone.cjs

```diff
@@ -299,11 +301,15 @@ async function standalone({
 
 /**
  * @param {FormatterType | Formatter | undefined} selected
+ * @param {string | undefined} configFormatter // Added parameter
  * @returns {Promise<Formatter>}
  */
-function getFormatterFunction(selected) {
-	if (typeof selected === 'string') {
-		const formatterFunction = index[selected];
+function getFormatterFunction(selected, configFormatter) {
+	// Modified function signature
+	const formatterToUse = selected || configFormatter; // Choose formatter from config if not specified
```

Comment:
minor
refactor

This should probably be done before calling this function.
i.e. this function should have only one argument

Reply:
Changed.